t/plugin/basic-auth.t

```diff
@@ -123,7 +123,10 @@ passed
                 ngx.HTTP_PUT,
                 [[{
                     "plugins": {
-                        "basic-auth": {}
+                        "basic-auth": {
+                            "username": "foo",
```

Comment:
That is wrong.

When a user requests, the authentication information is placed in an HTTP header.

Reply:
Hi @membphis , I'm a bit unclear, this test case is enabling basic auth plugin using admin api.

The test case 8 does the verification by adding the http header.

```

=== TEST 8: verify
--- request
GET /hello
--- more_headers
Authorization: Basic Zm9vOmJhcg==
--- response_body
hello world
--- no_error_log
[error]
```

I agree, in order to enable basic auth the admin does not have to send the same info again. Any suggestion how to tackle this would be helpful.  I guess the scenario is the same with key-auth plugin as well.